desktop/menu.ts

```diff
@@ -0,0 +1,103 @@
+import { app, Menu, MenuItemConstructorOptions, BrowserWindow } from 'electron'
+
+const template = [
+    {
+        label: app.name,
+        submenu: [
+            { role: 'about' },
+            { role: 'quit' }
+        ] as MenuItemConstructorOptions[]
```

Comment:
It's better to use type validation instead of type inforcement like:

```
const template: Electron.MenuItemConstructorOptions[] // or what is correct here
```

Otherwise using `as` will silently hide all configuration errors